ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -147,12 +158,14 @@ open class PillButton: UIButton {
         }
     }
 
-    private let unreadDotLayer: CALayer = {
-        let unreadDotLayer = CALayer()
-        unreadDotLayer.bounds.size = CGSize(width: Constants.unreadDotSize, height: Constants.unreadDotSize)
-        unreadDotLayer.cornerRadius = Constants.unreadDotSize / 2
-        return unreadDotLayer
-    }()
+    private lazy var unreadDotLayer: CALayer = {
```

Comment:
I know we spoke about this lazy instantiation of this CALayer in a previous iteration, but I just though of a scenario that should be considered:

Token update scenario:
Because its initialization reads from tokens that can be changed, this property needs to be recreated to reflect the tokens change.

And that also applies to the stackview property below in the bar.